### PR TITLE
Improve alignment of icons in the gas station

### LINKED ISSUE #3329: Loader in GasStation not vertically centered
## Steps to reproduce

1. Perform a transaction and watch the alignment of the loader in a transaction item.

## Expected behaviour

* Loader should be vertically centered

## Actual behaviour

* Loader is not vertically centered

download (36)

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Improved alignment of the loading, finished and group count icons that are in the gas station

FireShot Capture 586 - Mint 1000 PRR - Action - Colony - wonker - localhost
FireShot Capture 587 - Mint 1000 PRR - Action - Colony - wonker - localhost

Resolves #3329

### CODE REVIEW

**Review by arrenv — APPROVED** [review]
The alignment is looking much better to me.

8c605b2f-cf76-4153-b5ee-db3861cf17ca

While testing this I saw this alignment issue, are you seeing it as well?

f111e83b-3bf8-4013-a090-9e7404100175

File: src/modules/users/components/GasStation/TransactionCard/TransactionStatus.tsx [hunk 1]
```diff
@@ -70,7 +75,7 @@ const TransactionStatus = ({
           <FormattedMessage
             {...MSG.transactionState}
             values={{
-              status,
+              status: TRANSACTION_STATUSES.READY,
```
Comment (rdig) [inline src/modules/users/components/GasStation/TransactionCard/TransactionStatus.tsx:78]: I don't understand this change. Does this mean that from now on the status will always be set to `TRANSACTION_STATUSES.READY` ?

**Review by rdig — APPROVED** [review]
Looks good, left a comment.

Also, why is this merged into _Verified Recipients_ and not master ?

**Comment by ArmandoGraterol** [comment]
> While testing this I saw this alignment issue, are you seeing it as well?
> 
> f111e83b-3bf8-4013-a090-9e7404100175

@arrenv Seeing it as well! Gonna use this PR to fix that also

> Also, why is this merged into Verified Recipients and not master ?

@rdig A PR that fixes the alignment of the icons so that it's the same across all machines was merged into this branch (#3211), the icons in this PR must have been some that were leftover from the previous PR. 

The previous PR was merged into Verified Recipients because the feature had misaligned icons, so when fixing these, a solution for the rest was found :)

File: src/modules/users/components/GasStation/TransactionCard/TransactionStatus.tsx (hunk 1 above)
Comment (ArmandoGraterol) [reply src/modules/users/components/GasStation/TransactionCard/TransactionStatus.tsx:78]: Oof, my bad! This is a leftover from when I was testing the changes. 

Thank you for finding it! I thought I deleted them all.

**Pull request merged**